components/circle-progress.js

```diff
@@ -0,0 +1,58 @@
+import { Component } from 'react'
+import styled from 'styled-components'
+import { size } from 'polished'
+
+const Progress = styled.svg`
+  ${size('14em')}
+
+  fill: transparent;
+
+  circle {
+    stroke-linecap: round;
+    stroke-width: 10;
+    transform: translate(100px, 100px) rotate(-89.9deg);
+    transition: stroke-dashoffset 0.3s linear;
+    &.background {
+      stroke: ${props => props.theme.palette.borderColor};
+    }
+    &.progress {
+      stroke: ${props => props.theme.palette.primaryColor};
+    }
+  }
+
+  margin: auto;
```

Comment:
* Own styled components for `svg`, `circle` and `text`.
* At the moment we need no margin ...